limitador/src/storage/distributed/mod.rs

```diff
@@ -333,6 +334,15 @@ fn encode_counter_to_key(counter: &Counter) -> Vec<u8> {
 }
 
 fn encode_limit_to_key(limit: &Limit) -> Vec<u8> {
-    let counter = Counter::new(limit.clone(), HashMap::default());
+    // fixme this is broken!
```

Comment:
This has always been broken... and will need revisiting! But we can't just fake a the variables values like this (and they were simply left empty before, but this won't work now, as they'd fail resolution. I could "just" use the new `Counter::resolved_vars`, but that would just hide the problem a little more again.